Cargo.toml

```diff
@@ -16,6 +16,10 @@ rand = "0.4"
 byteorder = "1"
 clippy = { version = "0.0.200", optional = true }
 
+[dependencies.ff]
+git = "https://github.com/str4d/ff"
+rev = "91a8dc8284a09c456b36da44ca7de36f8b50550d"
+
 [features]
```

Comment:
Let's change `u128-support` feature in here to `u128-support = ["ff/u128-support"]` until we've removed `u128-support` from `ff`.